.github/workflows/ansible-test-integration.yml

```diff
@@ -3,16 +3,16 @@ on:
   # ! Don't enable pwn requests !
   # pull_request:
   push:
-    paths:
-      - 'plugins/**'
-      - 'tests/**'
-      - '.github/workflows/ansible-test-*.yml'
     branches:
-      - ci
+      - ci*
       - main
   schedule:
     - cron: '10 6 * * *'
 
+env:
```

Comment:
Move closer to https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml so it's easier to update in the future